### PR TITLE
Add linear elastic test 

### PR DESCRIPTION
**Pull request by ktbolt**
I've added a linear elasticity test case. See https://github.com/SimVascular/svMultiPhysics/issues/316.

### CODE REVIEW

File: tests/cases/linear-elasticity/beam/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+Solve a Eulear-Bernoulli beam problem with a traction boundary condition.
```
Comment (aabrown100-git) [inline tests/cases/linear-elasticity/beam/README.md:2]: Why are we calling this an Euler-Bernoulli beam problem? I think Euler-Bernoulli implies we're using [Euler-Bernoulli beam theory](https://en.wikipedia.org/wiki/Euler%E2%80%93Bernoulli_beam_theory), but this test case is solving standard 3D linear elasticity with a beam-shaped body.

File: tests/cases/linear-elasticity/beam/solver.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+</Add_mesh>
+
+
+<Add_equation type="lElas" > 
+
+   <Coupled> true </Coupled>
+   <Min_iterations> 1 </Min_iterations>  
+   <Max_iterations> 20 </Max_iterations> 
+   <Tolerance> 1e-5 </Tolerance>
```
Comment (aabrown100-git) [inline tests/cases/linear-elasticity/beam/solver.xml:60]: In other test cases, I think we're typically using a smaller tolerance (1e-9 or smaller).

File: tests/test_linear_elasticity.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from .conftest import run_with_reference
+import os
+import subprocess
+
+# Common folder for all tests in this file
+base_folder = "linear-elasticity"
+
+# Fields to test
+fields = [
```
Comment (aabrown100-git) [inline tests/test_linear_elasticity.py:9]: Are other fields available to be output for linear elasticity? It looks like we're testing all possible outputs for other physics.

**Review by aabrown100-git — CHANGES_REQUESTED** [review]

File: tests/cases/linear-elasticity/beam/README.md (hunk 1 above)
Comment (ktbolt) [reply tests/cases/linear-elasticity/beam/README.md:2]: @aabrown100-git This description was taken from `svFSI-Tests/02-lelas/01-beam_Euler-Bernoulli` which this test is based on. I assume it was called this because it reproduces the solution to the Euler-Bernoulli beam  equation.

File: tests/test_linear_elasticity.py (hunk 3 above)
Comment (ktbolt) [reply tests/test_linear_elasticity.py:9]: @aabrown100-git I will add strain and stress.

File: tests/cases/linear-elasticity/beam/solver.xml (hunk 2 above)
Comment (aabrown100-git) [reply tests/cases/linear-elasticity/beam/solver.xml:60]: What does TRT mean?

Comment (aabrown100-git) [reply tests/cases/linear-elasticity/beam/solver.xml:60]: I agree it's good to avoid confusion, but I worry that if tolerances are set too high, then our tests may not detect bugs introduced by future commits.

**Comment by mrp089** [comment]
Loosening (linear or nonlinear) tolerances can prevent us from detecting bugs or cause different results on different machines (that's why we made them stricter in existing tests).

We could introduce examples for more ready-to-use simulations. These could still be executed to ensure they always run, but not compare the results to a reference.

File: tests/cases/linear-elasticity/beam/solver.xml (hunk 2 above)
Comment (mrp089) [reply tests/cases/linear-elasticity/beam/solver.xml:60]: Loosening (linear or nonlinear) tolerances can prevent us from detecting bugs or cause different results on different machines (that's why we made them stricter in existing tests).

We could introduce examples for more ready-to-use simulations. These could still be executed to ensure they always run, but not compare the results to a reference.

**Comment by ktbolt** [comment]
@aabrown100-git Let's merge this.
